go.mod

```diff
@@ -1,28 +1,33 @@
-module github.com/worldcoin/semaphore-mtb-setup
+module github.com/mattstam/semaphore-mtb-setup
 
-go 1.19
+go 1.23
+
+toolchain go1.23.1
 
 require (
-	github.com/consensys/gnark v0.8.0
-	github.com/consensys/gnark-crypto v0.9.1
+	github.com/consensys/gnark v0.11.0
```

Comment:
after v10 didn't work (6GB+ solidity verifier), I tried rebuilding the circuits with v11 - but that also gave me a similar issue as the original time I tried this with a G1 index mismatch